manage.py

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python
+# lint-amnesty, pylint: disable=missing-docstring
```

Comment:
Just curious why the lint-amnesty is needed here?   Looks like a pretty short file, did it have errors?

Reply:
I think this can be removed, I decided to just add docstring to the file.